torchao/quantization/README.md

```diff
@@ -360,6 +360,36 @@ We have kernels that do 8-bit dynamic quantization of activations and uintx grou
 
 You try can out these apis with the `quantize_` api as above alongside the constructor `int8_dynamic_activation_intx_weight`.  An example can be found in `torchao/_models/llama/generate.py`.
 
+### Float8 `scaled_dot_product_attention` Support
```

Comment:
should this be in prototype?  The API doesn't match anything else we have in torchao, and in the PR summary you also mention that this isn't a final API.

Reply:
yeah this is prototype, this section is the prototype section, I can add a description on top

Reply:
it's not clear that this is prototype, the section title is "Other Available Quantization Techniques".  It would be great to clarify the expections on APIs in this section!